pkg/gitreceive/k8s_util.go

```diff
@@ -222,3 +226,22 @@ func waitForPodCondition(c *client.Client, ns, podName string, condition func(po
 		return false, nil
 	})
 }
+
+func progress(msg string) chan bool {
+	tick := time.Tick(8000 * time.Millisecond)
+	quit := make(chan bool)
+	go func() {
+		for {
+				select {
+				case <-quit:
+					close(quit)
+					return
+				case <-tick:
+                    fmt.Println(msg)
```

Comment:
Will the newlines show up in the `git` client? It would ruin the UX to have the useful log info scroll away off the page.